packages/quantic/cypress/e2e/default-1/refine-modal-content/refine-modal-content-expectations.ts

```diff
@@ -263,9 +264,50 @@ function refineContentExpectations(selector: RefineContentSelector) {
         })
         .logDetail('should order the facets correctly');
     },
+
+    refineSortOptionsEqual: (options: {value: string; label: string}[]) => {
+      options.forEach((option) => {
+        selector
+          .sortOption(option.value)
+          .should('exist')
+          .should('contain', option.label)
+          .logDetail(
+            'should contain the sort option ${option.label} in the refine modal sort component'
+          );
+      });
+    },
+  };
+}
+
+function sortExpectations(selector: SortSelector) {
+  return {
+    displaySort: (display: boolean) => {
+      selector
+        .sort()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the Quantic Sort component`);
+    },
```

Comment:
Same issue, this is using the same selector as `displayRefineModalSort` so how are we differentiating the sort outside the modal and inside the modal?

Reply:
@SimonMilord Still needs to be adressed.

Reply:
similar answer as [here](https://github.com/coveo/ui-kit/pull/4101#:~:text=its%20using%20the%20same%20selector%20but%20not%20the%20same%20root.%20The%202%20expectations%20use%20different%20selectors%20RefineContentSelector%20%26%20SortSelector%20so%20they%20are%20not%20targetting%20the%20same%20thing.%20I%20did%20essentially%20the%20same%20thing%20we%20do%20in%20the%20facets.)

Reply:
see in the selectors file im exporting "common" selectors for sorting just like the facets 
<img width="589" alt="image" src="https://github.com/user-attachments/assets/6e4f8821-b062-4079-b17b-b8cf3c8318aa">

Reply:
changed it to different selectors, they are now clearly for either refine modal or basic sort on the page